### ISSUE TITLE
Github gh-pages for project status overview (idea for pick-up)

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/semantic-mediawiki.org#7]
Maybe someone can pick up this task to create [0] for semanticmedia.github.io and have all repos with their badges listed to get an overview of the general project status.

[0] https://pages.github.com/

**Comment by kghbln** [comment]
Nice idea. I personally would like to have it at something like `status.semantic-mediawiki.org` to avoid having to remember yet another URL. At least we should link from such a subdomain to ...

**Comment by mwjames** [comment]
>  something like status.semantic-mediawiki.org to avoid having to remember ...

http://stackoverflow.com/questions/9082499/custom-domain-for-github-project-pages

**Comment by ivanhercaz** [comment]
Good idea! I accessed to [status.semantic-media.org](https://status.semantic-mediawiki.org) but I only see an IP and the validations for HTML5 and CSS.

Is this idea still in mind?

Regards,
Iván

**Comment by kghbln** [comment]
Yes. It is somewhere on the list.
